coral-component-masonry/src/scripts/Masonry.js

```diff
@@ -621,6 +621,10 @@ const Masonry = Decorator(class extends BaseComponent(HTMLElement) {
     } else {
       this.parentElement.removeAttribute('aria-colcount');
     }
+
+    if(this.parentElement.classList.contains('globalnav-collection-container')) {
```

Comment:
Coral-masonry is a generic cardview collection. Should not contain changes related to globalNav